packages/lab/src/Flow/stories/Flow.stories.tsx

```diff
@@ -195,3 +197,19 @@ export const BaseHook: StoryObj<HvFlowProps> = {
   },
   render: () => <BaseHookStory />,
 };
+
+export const SubFlow: StoryObj<HvFlowProps> = {
+  parameters: {
+    docs: {
+      description: {
+        story: "This sample demonstrate the use of a sub flow",
+      },
+      source: {
+        code: SubFlowRaw,
+      },
+    },
+    // Enables Chromatic snapshot
```

Comment:
Since this is a custom node, I think we can disable Chromatic for this sample. No need to keep track of visual changes in my opinion.

Reply:
Right, I only looked at the comment 😅 We could actually remove `chromatic: { disableSnapshot: true },` altogether since all samples are disabled by default.